### PR TITLE
[Poetry] Add spec for poetry

### PR DESCRIPTION
**Pull request by CastaChick**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
feature

**What is the current behavior? (You can also link to an open issue here)**
no spec for poetry

**What is the new behavior (if this is a feature change)?**
add new spec for poetry

**Additional info:**
poetry is package manager for Python

### CODE REVIEW

File: specs/fig.js [hunk 1]
```diff
@@ -102,7 +102,7 @@ var completionSpec = {
                     },
                 },
                 {
-                    name: "autocomplete.immediatelyExecute",
+                    name: "autocomplete.immediatelyExecuteAfterSpace",
```
Comment (QuiiBz) [inline specs/fig.js:105]: Why did you update this?

File: dev/poetry.ts [hunk 2]
```diff
@@ -0,0 +1,603 @@
...
+
+const noAnsi = {
+  name: ["--no-ansi"],
+  description: "Disable ANSI output",
+};
+
+const noInteraction = {
+  name: ["-n", "--no-interaction"],
+  description: "Do not ask any interactive question",
+};
```
Comment (QuiiBz) [inline dev/poetry.ts:44]: Could you add the `Fig.Option` type to these options?

File: dev/poetry.ts [hunk 3]
```diff
@@ -0,0 +1,603 @@
...
+        {
+          name: ["-E", "--extras"],
+          description: "Extras to activate for the dependency",
+          args: {
+            name: "extras",
+            description: "Extras",
+          },
+        },
+        {
+          name: ["--optional"],
```
Comment (QuiiBz) [inline dev/poetry.ts:88]: No need to specify an array, you can simply write it as a string since there is only one name. Same for underneath

**Review by QuiiBz — CHANGES_REQUESTED** [review]
Thanks for this big PR, I left a few improvements.

File: specs/fig.js (hunk 1 above)
Comment (CastaChick) [reply specs/fig.js:105]: When I create my feature branch, `specs/fig.js` and `dev/fig.ts` had different value.
It fixed by https://github.com/withfig/autocomplete/commit/a39c26e6f38a05b4b6df13ea78dcd38abd784d0b.

Could you show me how can I do it?

**Comment by CastaChick** [comment]
@QuiiBz Thanks for your review😃
I fixed requested changes!

File: specs/fig.js (hunk 1 above)
Comment (QuiiBz) [reply specs/fig.js:105]: You can update your branch to the latest Fig's master branch to get these changes.

Comment (CastaChick) [reply specs/fig.js:105]: I've fetched and merged the latest Fig's master branch.
However, the diff is still displayed. 
I tried to revert the commit, but it does not work well :(
Must I revert or squash some commits and force-push?

Comment (QuiiBz) [reply specs/fig.js:105]: You just need to merge `master` into `feat/add_poetry_spec` and push.

Comment (CastaChick) [reply specs/fig.js:105]: > You just need to merge master into feat/add_poetry_spec and push.
I did so, but it seems not reflected to this PR.

I tried reopening another PR from the same branch, the changed files are two.
Is it an issue of Github? Shall I reopen another PR?

Comment (QuiiBz) [reply specs/fig.js:105]: Strange, looks like it's an issue of GitHub. Could you open a new PR and close this one please? With a # to the new one.

Comment (CastaChick) [reply specs/fig.js:105]: I created another issue #201 ☺️

**Pull request closed without merging**